_pages/about-us/diversity.md

```diff
@@ -34,14 +34,14 @@ Find us in Slack:
 
 _It&rsquo;s important that the responsibility of answering questions around diversity and inclusion doesn&rsquo;t fall on one or two people. Our Slack channel is open space in which our colleagues (who genuinely want to do the right thing) can ask hard questions of one another. It allows us to collectively place the responsibility of diversity and inclusion on multiple people rather than any one person._
 
-### <a id="philosophy">Philosophy</a>
+### Philosophy
 
 The Diversity Guild is a space for learning. If you make a mistake &mdash; and we all do &mdash; a key component of diversity is that we have a culture of mindfulness and trying to do better. Remaining open to feedback, learning about why an action was offensive, and then apologizing is the kind of environment we look to cultivate.
 
 The guild realizes that building an inclusive environment is tough, but it&rsquo;s far from impossible. We base our work on actionable items and deliverables which are reflected in our [Outcomes and Key Results (OKRs)](https://docs.google.com/a/gsa.gov/document/d/1bXXVpGE0OGFTJHQklo4k7-M83dA4RQqvN5qIGklzh1g/edit?usp=sharing). These OKRs are discussed and decided every quarter.
 
 
-### <a id="trello">Tracking progress on Trello</a>
+### Tracking progress on Trello
```

Comment:
I think in this case, an author wanted manual control, because they wanted to anchor to `#trello` not `#tracking-progress-on-trello` (or whatever Kramdown would create from this header. 

Personally, I prefer the shorter, manually controlled ID, but if we do want to remove it, I think we'll have to update the nav linking to it.